examples.conf.template

```diff
@@ -0,0 +1,16 @@
+# Properties prefixed with accumulo.examples are not general Accumulo properties
```

Comment:
Not sure it matters too much. I prefer `.properties`, over `.conf`, but it that could also imply that you can use `-D` for these same Java properties on the command-line, and you probably can't here (though, it'd be nice if you could).